### PR TITLE
Collections page UI changes

### PR DESCRIPTION
**Pull request by daltojohnso**
Related to #19758 

Bunch of UI tweaks to the Collections page

- the collection description is shown now
- some background colors have changed
- the pin action is exposed for unpinned items

Screen Shot 2022-01-19 at 7 40 17 PM

### CODE REVIEW

File: frontend/src/metabase/components/EntityItem.jsx [hunk 1]
```diff
@@ -105,17 +109,29 @@ function EntityItemMenu({
...
     return null;
   }
   return (
-    <EntityMenu
-      className={cx(className, "hover-child")}
-      triggerIcon="ellipsis"
-      items={actions}
-    />
+    <EntityMenuContainer align="center">
+      {!isPinned && <PinButton icon="pin" onClick={onPin} />}
```
Comment (mazameli) [inline frontend/src/metabase/components/EntityItem.jsx:128]: This could use a `Pin this` tooltip on hover.

Comment (daltojohnso) [reply frontend/src/metabase/components/EntityItem.jsx]: Done

File: frontend/src/metabase/collections/components/BaseItemsTable.jsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import {
   Table,
   SortingIcon,
   SortingControlContainer,
+  Tbody,
```
Comment (alxnddr) [inline frontend/src/metabase/collections/components/BaseItemsTable.jsx:11]: nit: TBody would be more conventional

**Review by alxnddr — COMMENTED** [review]
Great job! I love the new look and feel.

A couple of questions:
1) Should we align cards with different heights without a larger gap?
Screenshot 2022-01-21 at 14 50 06

2) Should the DnD work for pinned items?

**Comment by daltojohnso** [comment]
@alxnddr: 

1. good catch; the awkward gap isn't ideal. We can probably add a bit of JS to calculate the rows of the grid, but I'll leave that for a second pass. FYI @mazameli 
2. DND for pinned cards is in a separate PR -- https://github.com/metabase/metabase/pull/19829

**Review by alxnddr — APPROVED** [review]

**Comment by daltojohnso** [comment]
Yep I can throw a tooltip on it like with the description, if you'd like

**Comment by daltojohnso** [comment]
@mazameli  -- this is the behavior now:

https://user-images.githubusercontent.com/13057258/150578832-c4e02730-b67e-43a0-8bef-21b409806d41.mov

**Pull request merged**